packages/components/src/palette-edit/index.js

```diff
@@ -101,6 +101,11 @@ function ColorPickerPopover( {
 			offset={ 20 }
 			className="components-palette-edit__popover"
 			onClose={ onClose }
+			aria-label={
+				! isGradient
+					? __( 'Select a new color' )
+					: __( 'Select a new gradient' )
+			}
```

Comment:
I don't know too much about `aria-label` usage, but from a quick look at [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label), it sounds like it's intended to be used only on interactive elements (to indicate how an element can be interacted with) rather than to wrappers?

I was wondering if the addition here is so that the tests have something to hook onto for determining whether or not `ColorPicker` or `CustomGradientPicker` is rendered. If so, could it would work to look up the label "Color" for the color picker, or the label (or part of the label) `Gradient control point at position" for the custom gradient picker?

Markup for color picker:

![image](https://user-images.githubusercontent.com/14988353/205197327-22f63404-710e-4471-8122-b65ed51707c7.png)

Markup for gradient picker:

<img width="649" alt="image" src="https://user-images.githubusercontent.com/14988353/205197246-25ab239d-cb24-4aed-afaf-55b053f9264a.png">

Reply:
You're right. Excellent point. Thanks for making this PR better @andrewserong !